app/src/app/list/list.component.html

```diff
@@ -1,9 +1,15 @@
 <app-server-error [error]="error$ | async"></app-server-error>
-<div class="poll-list" *ngIf="(publicList$ | async)?.length">
-  <h3>Polls you participate in</h3>
-  <app-polls-table [polls]="publicList$ | async"></app-polls-table>
-</div>
-<div class="poll-list" *ngIf="(ownList$ | async)?.length">
-  <h3>Polls you have created</h3>
-  <app-polls-table [polls]="ownList$ | async"></app-polls-table>
-</div>
+<mat-tab-group mat-align-tabs="center" id="tab-polls">
+  <mat-tab  label="Participate" class="poll-list" *ngIf="(publicList$ | async)?.length">
+    <div class="poll-list">
+      <h3>Polls you participate in</h3>
+      <app-polls-table [polls]="publicList$ | async"></app-polls-table>
+    </div>
+  </mat-tab>
+  <mat-tab label="Created" class="poll-list" *ngIf="(ownList$ | async)?.length">
+    <div class="poll-list">
+      <h3>Polls you have created</h3>
+      <app-polls-table [polls]="ownList$ | async"></app-polls-table>
+    </div>
+  </mat-tab>
+</mat-tab-group>
```

Comment:
A newline is missing here. Ending all lines of text files with a newline is a widely adopted convention. Please configure your text editor correctly and add a newline at the end of this file. Do the same for list.component.sass.